spec/controllers/idv_controller_spec.rb

```diff
@@ -86,7 +86,7 @@
         stub_sign_in(profile.user)
       end
 
-      it 'redirects to rate limited page' do
+      it 'redirects to failure page' do
```

Comment:
During review, I noticed that 'rate_limited' is the doc_auth rate limited url, and 'failure' is the resolution rate limited url, so this spec title should remain unchanged. I'll make a typo-fix commit and include this.